packages/react-native-renderer/src/ReactNativeAttributePayloadFabric.js

```diff
@@ -462,27 +462,27 @@ function fastAddProperties(
 
     attributeConfig = validAttributes[propKey];
 
-    if (attributeConfig === undefined) {
+    if (!attributeConfig) {
       continue;
     }
```

Comment:
What I was just mentioning about: we could change the default here and stop ignoring any properties that do not have `validAttributes` entry.

Reply:
That requires the new parser, no? Otherwise, we might get a native exception.

Reply:
Update native to ignore unknown props?

This would take us a step closer to removing Static View Configs. We would need this, and a way to specify the bubbling behavior of events. Or we could convert all custom events to be direct, and store a config for which built-in events should bubble (which would also fix the bubbing/direct event name collision bug).

Reply:
> Update native to ignore unknown props?
> This would take us a step closer to removing Static View Configs.

That is already planned!